### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by loremattei**
This PR updates the main README.md and adds a short contributing guide (from the WordPress apps) and a second README in the `actions` folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+If you have questions about getting setup or just want to say hi, join the [WordPress Slack](https://chat.wordpress.org) and drop a message on the `#mobile` channel.
+
+## Resources
+
+- [WordPress Mobile Blog](http://make.wordpress.org/mobile)
+- [WordPress Mobile Handbook](http://make.wordpress.org/mobile/handbook/)
+
+## License
+
+WordPress for iOS is an Open Source project covered by the [GNU General Public License version 2](LICENSE).
```
Comment (elibud) [inline README.md]: Should this be the Mobile Release Toolkit?

File: lib/fastlane/plugin/wpmreleasetoolkit/actions/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+The Actions in this library are organized in 4 folders:
+
+### Android
+
+Actions in this folder automate some tasks of the WordPress for Android release process. 
+
+### Common
+
+Actions in this folder automate tasks that are common to both the iOS and Android release processes.
+These actions are cross-platforms and usually related to interating with GitHub, GlotPress and other services.
```
Comment (elibud) [inline lib/fastlane/plugin/wpmreleasetoolkit/actions/README.md]: 'interating' should be 'integrating'?

File: README.md (hunk 1 above)
Comment (loremattei) [reply README.md]: Yep! Thank you!

File: lib/fastlane/plugin/wpmreleasetoolkit/actions/README.md (hunk 2 above)
Comment (loremattei) [reply lib/fastlane/plugin/wpmreleasetoolkit/actions/README.md]: You're right! Thanks for catching it!

**Pull request merged**